controllers/upgrade.go

```diff
@@ -634,3 +634,29 @@ func (r *RollingUpgradeContext) endTimeUpdate() {
 		common.TotalProcessingTime(r.RollingUpgrade.ScalingGroupName(), totalProcessingTime)
 	}
 }
+
+// AWS API call for setting an instance to StandBy has a limit of 19. Hence we have to call the API in batches.
+func (r *RollingUpgradeContext) SetBatchStandBy(instanceIDs []string) error {
+	var (
+		pendingInstances = len(instanceIDs)
+		numberOfBatches  = math.Ceil(float64(len(instanceIDs)) / float64(awsprovider.InstanceStandByLimit))
```

Comment:
This should be round up, right ?
Let's say   65 instances,   limit is 20,  it requires  4 batches

Reply:
60 instances requires 3 batches